src/functions/functions-api.ts

```diff
@@ -58,6 +60,40 @@ export type TaskOptions = DeliverySchedule & TaskOptionsExperimental & {
    * The default is 10 minutes. The deadline must be in the range of 15 seconds and 30 minutes.
    */
   dispatchDeadlineSeconds?: number;
+
+  /**
+   * The id to use for the enqueued event.
+   * If not provided, one will be automatically generated.
+   * If provided explicitly specifying a task ID enables task de-duplication. If a task's ID is
```

Comment:
iiuc, this reads better as, "If provided, an explicitly specified task ID enables . . ."